app/assets/stylesheets/bubble.scss

```diff
@@ -0,0 +1,50 @@
+$white: #ffffff;
+$oil: #333333;
+
+.bubble {
+  position: relative;
+  width: auto;
+  min-height: 200px;
+  padding: 5px;
+  background: $white;
+  -webkit-border-radius: 18px;
+  -moz-border-radius: 18px;
+  border-radius: 18px;
+  border: #FFFFFF solid 2px;
+  -moz-box-shadow:  -2px 2px 2px 0px rgba(67, 65, 65, 0.99);
+  -webkit-box-shadow:  -2px 2px 2px 0px rgba(67, 65, 65, 0.99);
+  box-shadow:  -2px 2px 2px 0px rgba(67, 65, 65, 0.99);
+  margin: 26px auto 0;
+
+  li {
+    color: $oil;
+    font-weight: 500;
+    list-style: none;
+  };
+}
+
+.bubble:after {
+  content: '';
+  position: absolute;
+  border-style: solid;
+  border-width: 10px 0 10px 13px;
+  border-color: transparent #FFFDFF;
+  display: block;
+  width: 0;
+  z-index: 1;
+  right: -13px;
+  top: 48px;
+}
+
+.bubble:before {
+  content: '';
```

Comment:
Properties should be ordered border-color, border-style, border-width, content, display, position, right, top, width, z-index<br>Prefer double-quoted strings